CardParts/src/Extensions/UIFont.swift

```diff
@@ -43,4 +43,6 @@ extension UIFont {
     class func turboGenericLightFontWithSize(_ size: CGFloat) -> UIFont {
```

Comment:
More places that I want to remove and make more generic.  Should we make it not `turboGenericLightFontWithSize ` but instead `lightFontWithSize`

Reply:
yes we should make it to `lightFontWithSize`

Reply:
Can we do it as part of next PR's ?